src/guides/v2.4/graphql/mutations/update-customer.md

```diff
@@ -65,6 +65,17 @@ The following table lists the top-level attributes of the `customer` object. See
 
 {% include graphql/customer-output-24.md %}
 
+## Errors
+
+Error | Description
+--- | ---
+`"input" value should be specified` | The `input` argument is empty.
+`"Email" is not a valid email address.` | The value provided in the `input`.`email` argument has an invalid format.
+`Provide the current "password" to change "email".` | To change an email address, specify the correct customer password in the `password` argument.
+`A customer with the same email address already exists in an associated website.` | You cannot set a new email to a current customer, because another user has the same email address.
```

Comment:
```suggestion
`A customer with the same email address already exists in an associated website.` | You cannot apply a new email address to a current customer because another user has the same email address.
```